pkg/apiserver/registries/registries.go

```diff
@@ -68,6 +69,25 @@ func RegistryImageBlob(request *restful.Request, response *restful.Response) {
 		return
 	}
 
+	// default use ssl
+	checkSSl := func(serverAddress string) bool {
```

Comment:
check the schema here, and do another check on schema. I think the first thing here is to properly parse image address, then continue to work on this parse result, this will save lots of schema checking. We can do this refactoring later on version 3.0, but keep in mind.

Reply:
before parse image, should remove protocol. if parse image address first with protocol,  will get an error.